.github/workflows/subscribe-to-mns.yml

```diff
@@ -16,18 +16,17 @@ on:
       AWS_ASSUME_ROLE:
         required: true
 permissions:
-  pull-requests: write
   id-token: write # This is required for requesting the JWT
   contents: read # This is required for actions/checkout
 jobs:
-  batch_update_build_docker_image:
+  placeholder:

```

Comment:
Do we still want this to be a placeholder?

Reply:
the workflow that we want is currently on a different ticket that hasn't yet been merged in.